pkg/pods/pod.go

```diff
@@ -641,3 +653,8 @@ func (pod *Pod) csvHeaders() string {
 
 	return pod.fqCsvHeaders
 }
+
+func init() {
```

Comment:
Where is `init()` called? I would consider restructuring this to move from pods to a separate name or text validator package, as we also know we need to validate pod name, and probably other strings too.

Reply:
Moved to its own `validator` package